db/migrate/20180314090401_move_column_data_to_authorizations_table.rb

```diff
@@ -0,0 +1,10 @@
+class MoveColumnDataToAuthorizationsTable < ActiveRecord::Migration[5.1]
+  def change
+    User.where.not(category: [nil, '']) do |user|
```

Comment:
@sinpai wh do you check whether `user` has categories? Shouldn't we migrate data of all users?